doc/documentation-main.html

```diff
@@ -1793,6 +1793,52 @@ <h4><a class="section" indent=".." name="rerunning">Rerunning failed tests</a></
 methods so that you are guaranteed to run the methods that failed without any 
 SKIP failures.</p>
 
+Sometimes you might also want TestNG to automatically retry a test whenever it fails. In those situations you can make use of a retry analyzer.
+When you bind a retry analyzer to a test, TestNG automatically invokes the retry analyzer to determine if TestNG can retry a test-case
+once again in an attempt to see if the flaky test passes. Following is how you can use a retry analyzer.
+
+<ol>
+  <li>Build an implementation of the interface <tt>org.testng.IRetryAnalyzer</tt></li>
+  <li>Bind this implementation to the <tt>@Test</tt> annotation for e.g., <tt>@Test(retryAnalyzer = LocalRetry.class)</tt></li>
+</ol>
+
+Following is a sample implementation of the retry analyzer that retries a test for a maximum of 3 times.
```

Comment:
"three" times (spell out small numbers).

Reply:
Amended the commit and fixed as suggested